### PR TITLE
f-checkout@0.5.0 - Add f-checkout component tests

### PR DESCRIPTION
**Pull request by siggerzz**
### Added
- New functionality to component object
- New component tests

### CODE REVIEW

File: packages/f-checkout/test-utils/component-objects/f-checkout.component.js [hunk 1]
```diff
@@ -1,4 +1,68 @@
...
+
+const userNoteInput = () => $('[data-test-id="user-note"] textarea');
+
+const allergenLink = () => $('[data-test-id="allergy-button"]');
+
+const deliveryTimeDropdown = () => $('[data-test-id="delivery-time"]');
+
+const goToPaymentButton = () => $('[data-test-id="confirm-payment-submit-button"]');
+
 exports.waitForCheckoutComponent = () => checkoutComponent().waitForExist();
```
Comment (LTurns) [inline packages/f-checkout/test-utils/component-objects/f-checkout.component.js:18]: more a q than a comment - do we need a test that the whole component is displayed if by default it must be when we test the other elements inside of it? Noticed we didn't do that for f-footer but happy to add in!

Comment (siggerzz) [reply packages/f-checkout/test-utils/component-objects/f-checkout.component.js:18]: `waitForExist()` only checks for presence in the DOM, not that it's necessarily displayed, so I'd still have tests for visibility :)

File: packages/f-checkout/test/specs/component/f-checkout.component.spec.js [hunk 2]
```diff
@@ -4,10 +4,35 @@ describe('f-checkout component tests', () => {
...
         // Arrange
         browser.url('http://localhost:8080');
+        CheckoutComponent.waitForCheckoutComponent();
     })
 
     it('should display the f-checkout component', () => {
         // Assert
         expect(CheckoutComponent.isCheckoutComponentDisplayed()).toBe(true);
     });
+
```
Comment (LTurns) [inline packages/f-checkout/test/specs/component/f-checkout.component.spec.js:14]: missing a few semi-colons if I'm nit-picking, but looks like we're waiting for an assert anyway

File: packages/f-checkout/test-utils/component-objects/f-checkout.component.js (hunk 1 above)
Comment (LTurns) [reply packages/f-checkout/test-utils/component-objects/f-checkout.component.js:18]: ah okay awesome, makes sense!

**Review by LTurns — DISMISSED** [review]

**Review by simonsandersje — DISMISSED** [review]

File: packages/f-checkout/src/components/Selector.vue [hunk 3]
```diff
@@ -67,7 +65,6 @@ $form-input-borderColour--focus           : $grey--dark;
     background-color: $form-input-bg;
     border: $form-input-borderWidth solid $form-input-borderColour;
     border-radius: $form-input-borderRadius;
-
```
Comment (ashleynolan) [inline packages/f-checkout/src/components/Selector.vue]: Good idea to keep these line breaks – makes the styles more readable (not sure if this is being auto-formatted by your code editor?)

Comment (siggerzz) [reply packages/f-checkout/src/components/Selector.vue]: Yeah, this was the result of a bad merge conflict. Will sort this out 👍

**Review by simonsandersje — APPROVED** [review]

**Review by LTurns — APPROVED** [review]

**Pull request merged**